src/main/kotlin/racing/model/Racing.kt

```diff
@@ -1,5 +1,23 @@
 package racing.model
 
-class Racing(carCount: Int) {
-    var cars: List<Car> = List(carCount) { Car() }
+class Racing(carNames: List<String>, val tryGameCount: Int) {
+    var cars: MutableList<Car> = mutableListOf()
+
+    init {
+        for (carName in carNames) {
+            cars.add(Car(carName))
+        }
+    }
+
+    fun play() {
+        repeat(tryGameCount) {
+            cars.forEach { car ->
+                val moving = car.run()
+                print("${car.name} : ")
+                println(Car.CAR_SYMBOL.repeat(moving))
```

Comment:
람다 변수가 하나뿐이라면 람다 변수 선언 없이 `it`으로 대체해서 사용할 수 있어요. 😉
아래와 같이 바꿔볼 수 있을 것 같아요. 😃
```suggestion
            cars.forEach {
                val position = it.run()
                println("${car.name} : ${Car.CAR_SYMBOL.repeat(position)}")
```

Reply:
이 부분은 5단계 미션의 요구사항인데요. 😃 
이 Racing 객체의 `play` 함수 내부의 print 문만 해결한다면, MVC 패턴을 지킬 수 있을 것 같아요. 👀
다음 단계에서는 이 부분을 개선해보시면 좋을 것 같아요. 😉